backend/src/middleware/slugifyMiddleware.spec.js

```diff
@@ -1,81 +1,62 @@
+import { GraphQLClient } from 'graphql-request'
 import Factory from '../seed/factories'
-import { gql } from '../jest/helpers'
-import { neode as getNeode, getDriver } from '../bootstrap/neo4j'
-import createServer from '../server'
-import { createTestClient } from 'apollo-server-testing'
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-09T13:05:41Z" title="Monday, September 9th 2019, 3:05:41 pm +02:00">Sep 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -89,8 +89,8 @@ describe('slugify', () => {
   })
 
   describe('SignupVerification', () => {
-    const mutation = `mutation($password: String!, $email: String!, $name: String!, $slug: String, $nonce: String!, $termsAndConditionsAgreedVersion: String!) {
-      SignupVerification(email: $email, password: $password, name: $name, slug: $slug, nonce: $nonce, termsAndConditionsAgreedVersion: $termsAndConditionsAgreedVersion) { 
+    const mutation = `mutation($password: String!, $email: String!, $name: String!, $slug: String, $nonce: String!, $termsAndConditionsAgreedVersion: String!, termsAndConditionsAgreedAt: String) {
```


No, the `termsAndConditionsAgreedAt` should *not* be supplied by the user. We don't care what the user tells us, what date it is. We **know** the date *on the server*. So, in the `SignupVerification` resolver, we set the `termsAndConditionsAgreedAt` manually.